vm/Cargo.toml

```diff
@@ -23,6 +23,7 @@ regex = "1"
 rustc_version_runtime = "0.1.*"
 statrs = "0.10.0"
 caseless = "0.2.1"
+unicode-casing = { git = "https://github.com/OddCoincidence/unicode-casing" }
```

Comment:
Could you create a release from this package, so we can refer to a pinned version?

Reply:
You mean on crates.io? I was going to see if the rust unicode folks wanted control over this package before submitting. It's fine if you want to wait for that, but I can also add the commit sha here if that makes you feel better.

Reply:
Ah, okay, yes, rust unicode group having this package would be good. Let's keep it like this, until this is effect. Maybe add a todo?